packages/aft/lib/src/commands/generate/generate_workflows_command.dart

```diff
@@ -219,4 +223,90 @@ jobs:
 
     androidWorkflowFile.writeAsStringSync(androidWorkflowContents);
   }
+
+  /// If a package has iOS unit tests, generate a separate workflow for them.
+  Future<void> generateIosUnitTestWorkflow({
+    required PackageInfo package,
+    required String repoRelativePath,
+  }) async {
+    const iosWorkflow = 'flutter_ios.yaml';
+    final platformPackageName = '${package.name}_ios'; // federated _ios package
+    final platformPackagePath = '${package.path}_ios';
+    final appFacingIosTestDir =
+        Directory(p.join(package.path, 'example', 'ios', 'unit_tests'));
+    final platformIosPackageDir = Directory(platformPackagePath);
```

Comment:
Naming changed from without `ios` to with `ios`. I would prefer without I think, since we're already in an iOS context. But either way - just better to be consistent.